lib/ansible/galaxy/dependency_resolution/__init__.py

```diff
@@ -21,15 +21,15 @@
 
 
 def build_collection_dependency_resolver(
-        galaxy_apis,  # type: t.Iterable[GalaxyAPI]
-        concrete_artifacts_manager,  # type: ConcreteArtifactsManager
-        preferred_candidates=None,  # type: t.Iterable[Candidate]
-        with_deps=True,  # type: bool
-        with_pre_releases=False,  # type: bool
-        upgrade=False,  # type: bool
-        include_signatures=True,  # type: bool
-        offline=False,  # type: bool
-):  # type: (...) -> CollectionDependencyResolver
+        galaxy_apis: t.Iterable[GalaxyAPI],
```

Comment:
The collection types in the `typing` module are deprecated. Use the ones in `collections.abc` instead.

This is the first occurrence I spotted, but there are more throughout this PR.

Reply:
Done locally. Marking this thread as resolved.